### DISCUSSION TITLE
automazione condizionata da z2m

### DISCUSSION THREAD

**Discussion by ster1um** [indomus/forum#2470] [HUB: Home Assistant]
è possibile inserire una condizione che faccia intervenire l’automazione soltanto quando il comando arriva da z2m?

In pratica devo impostare una soglia di luminosità quando l’input accensione arriva da z2m ma non da es. homekit, alexa etc.

**Comment by snakuzzo** [comment]
Se il device è esposto da zigbee2mqtt verso HA e poi verso Homekit e Alexa, quello che chiedi credo non si possa fare...questo perché di fatto sarà sempre pilotato tramite zigbee2mqtt.

Unica cosa che mi viene in mente è andare a beccare gli eventi, tradurli e cercare di fare un match con l'entità che andresti a pilotare...ma credimi...sarebbe davvero troppo complesso e contorto che avresti sicuramente risultati inattesi

**Reply by ster1um** [reply]
ok ti ringrazio, già avevo capito la complessità della cosa ma per sicurezza ho preferito chiedere magari c'era un escamotage per poterlo effettuare.
